examples/generic-node-example/package.json

```diff
@@ -6,15 +6,17 @@
   "license": "MIT",
   "dependencies": {
     "ethers": "5.7.2",
-    "@buildwithsygma/sygma-sdk-core": "*",
+    "@buildwithsygma/sygma-sdk-core": "^1.1.4",
```

Comment:
Use monorepo version not external

Reply:
monorepo version is `"*"`?

Reply:
I will rebase this PR with the latest from main that includes latest version of yarn and all examples are using monorepo version of sdk